### ISSUE TITLE
Make no-mixed-operators rule fixable

### ISSUE THREAD

**Issue by medikoo** [eslint/eslint#8916]
Adding parens won't change the behavior of the code,  additionally as a POC [prettier](https://github.com/prettier/prettier) already auto wraps logical operators.

---

__What rule do you want to change?__

`no-mixed-operators`

__Does this change cause the rule to produce more or fewer warnings?__

No

__How will the change be implemented? (New option, new default behavior, etc.)?__

_Fix_ functionality will be configured

__Please provide some example code that this change will affect:__

Code as:

```javascript
if (a * b + c) return d;
```

Will be fixed to:

```javascript
if ((a * b) + c) return d;
```

__What does the rule currently do for this code?__

There's no _fix_ functionality configured

__What will the rule do after it's changed?__

_Fix_ functionality will be configured

**Comment by not-an-aardvark** [comment]
Thanks for the proposal. I'm a bit concerned that this would end up concealing errors if the user made a mistake with precedence and intended for the operators to be grouped in some other way, but I'm unsure how often that would actually occur.

**Comment by not-an-aardvark** [comment]
Thanks for your interest in improving ESLint. Unfortunately, it looks like this issue didn't get consensus from the team, so I'm closing it. We define [consensus](http://eslint.org/docs/maintainer-guide/issues.html#consensus) as having three :+1:s from team members, as well as a team member willing to champion the proposal. This is a high bar by design -- we can't realistically accept and maintain every feature request in the long term, so we only accept feature requests which are useful enough that there is consensus among the team that they're worth adding.

Since ESLint is pluggable and can load custom rules at runtime, the lack of consensus among the ESLint team doesn't need to be a blocker for you using this in your project, if you'd find it useful. It just means that you would need to [implement the rule](http://eslint.org/docs/developer-guide/working-with-rules) yourself, rather than using a bundled rule that is packaged with ESLint.
